acknowledge_license_cert.html

```diff
@@ -11,28 +11,35 @@
 
 <h3>License information</h3>  
 <div class="paragraph">   
-Unless otherwise stated, this code is subject to the terms of the <a href="license.txt">
-GNU General Public License version 3 or higher (GPL)</a>, a copy of which is included here
- for your convenience.
+To be announced
```

Comment:
Changed

      From: Tony McCormick <notifications@github.com>
 To: LibreEHR/LibreEHR <LibreEHR@noreply.github.com> 
Cc: Terry Hill <teryhill@yahoo.com>; Author <author@noreply.github.com>
 Sent: Thursday, August 11, 2016 8:19 PM
 Subject: Re: [LibreEHR/LibreEHR] Changes to the acknowledgments page (#80)
   
In acknowledge_license_cert.html:> @@ -11,28 +11,35 @@
>  
>  <h3>License information</h3>  
>  <div class="paragraph">   
> -Unless otherwise stated, this code is subject to the terms of the <a href="license.txt">
> -GNU General Public License version 3 or higher (GPL)</a>, a copy of which is included here
> - for your convenience.
> +To be announced
I think you have to say that the code is GPL, since it is and we can't change that with out the authors permission. In the future we can add a line about the code that is Apache or whatever it ends up being—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.